testing/run-scripts/run_cloud.sh

```diff
@@ -188,4 +188,20 @@ if ! docker ps -a | grep uproxy-sshd >/dev/null; then
   fi
 fi
 
+echo "CLOUD_INSTALL_PROGRESS 95"
+
+# Boot monitoring servers and metrics.
```

Comment:
Until this stuff is a little more mature, I think we should move this to a separate script. How about `run_monitor.sh`? It should be run after `run_cloud.sh`.